WordPress/src/main/res/drawable/ic_tick_vector_blue.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#3C5ACB"
```

Comment:
We can add this color to `colors.xml` or check if the desired color is already available there.

Reply:
Thank you for making this change 👍